### DISCUSSION TITLE
`ComboBox` does not show more selected element when width is increased

### DISCUSSION THREAD

**Discussion by saskliutas** [iTwin/iTwinUI#2358] [General]
I noticed that if I have `ComboBox` with multiple selected values and there is not enough space to show all value some of them are hidden under overflow tag (+x item(s)). However if width is increased and more items are not shown. For example:
![ComboBoxOverflow](https://github.com/user-attachments/assets/3b3d9773-3fab-4b1b-b723-13d1c6ceb019)

Link to sandbox: https://stackblitz.com/edit/github-tzuyww?file=src%2FApp.tsx
I was wondering if this is working as designed?

**Comment by mayank99** [comment]
Thanks for checking. This is a known issue which we decided not to handle in our recent refactored overflow algorithm. It was considered to be an edge case because users wouldn't usually resize the window in the middle of operating the combobox.
